src/primitives/block.h

```diff
@@ -89,6 +90,23 @@ class CBlock : public CBlockHeader
         *(static_cast<CBlockHeader*>(this)) = header;
     }
 
+    CBlock(const CBlock &block)
+        :CBlock(static_cast<CBlockHeader>(block))
+    {
+        vtx = block.vtx;
+        fChecked = false;
```

Comment:
Any reason to set this to false when copying?

nit: Also, could rename the member to `m_checked`, since it is only used in one function on master.